### ISSUE TITLE
Vertical Rotation of the Ship Placement

### ISSUE THREAD

**Issue by peterkyl** [cs361fall2018/project-team-36#21]
I swear this worked the other day, worth investigating.

**Comment by peterkyl** [comment]
Mostly fixed in #23. The ships now highlight multiple spaces when both horizontal and vertical but if you switch horizontal->vertical or vice versa while hovering over the grid it will shade in some of the grid permanently, until the board is redrawn.

**Comment by peterkyl** [comment]
Fixed this issue in #27 

The problem is that if you rotate while the cursor is on the grid it will toggle some cells only once instead of twice like normal. So I added to the keyup function, which handles input from the keyboard and toggles the vertical status, the ability to clear the 'placed' class which is what the gray boxes are. However this introduced the problem that some boxes are supposed to still have the placed status so then I added code copied from the place function (calling place directly wont work because place assumes its called as an event listener on the table). This re toggles the class on the correct cells.

The current way rotating works: The vertical checkbox, when checked, strictly enforces vertical. When unchecked the 'r' key will toggle vertical between true and false (vertical = !vertical).

We need to decide what functionality we should provide our users, do we remove the checkbox? Do we add scroll wheel as an input to toggle rotation? Or some other idea.
